src/cli/services/__tests__/linter.test.ts

```diff
@@ -607,22 +607,6 @@ describe('Linter service', () => {
           },
           source: expect.stringContaining('__tests__/__fixtures__/refs/info.json'),
         }),
-        expect.objectContaining({
-          code: 'oas2-schema',
-          message: '`description` property type should be string.',
-          path: ['definitions', 'info', 'description'],
-          range: {
-            end: {
-              character: 22,
-              line: 5,
-            },
-            start: {
-              character: 21,
-              line: 5,
-            },
-          },
-          source: expect.stringContaining('__tests__/__fixtures__/refs/info.json'),
-        }),
```

Comment:
These two changes are another instance where the `required` error is no longer filtered out by `prepareResults` and that drives `better-ajv-errors` to filter out all of its "child" errors.